pipelines/LSSTComCam/cpDefects.yaml

```diff
@@ -1,4 +1,29 @@
-description: cp_pipe DEFECT calibration construction from combined images.
+description: cp_pipe LSSTComCam defect calibration construction.
 instrument: lsst.obs.lsst.LsstComCam
 imports:
-  - location: $CP_PIPE_DIR/pipelines/_ingredients/cpDefects.yaml
+  - location: $CP_PIPE_DIR/pipelines/_ingredients/cpDefectsLSST.yaml
+tasks:
+  cpMeasureBiasDefects:
+    class: lsst.cp.pipe.MeasureDefectsCombinedTask
+    config:
+      badPixelsToFillColumnThreshold: 1000
```

Comment:
Do we expect these config options (for all three tasks) to differ between the three cameras?

Reply:
I have left them the same for now, because of cargo culting.  But I could see these being different for LSSTCam based on defect research.